setup.py

```diff
@@ -59,7 +59,7 @@ def main():
     setup(**skw)
 
 
-    if platform.system() == 'Darwin':
+    if platform.system().lower() == 'darwin':
```

Comment:
this may also be too specific.  Is it just Darwin that is affected?  We could make this a list `PW_AFFECTED_OSX_SYSTEMS = ["darwin"]`  then `if platform.system().lower() in PW_AFFECTED_OSX_SYSTEMS`, something like that.  We should only add systems that are affected, so find as many as can be tested, but if people complain to diffpy-users we can add more.   For now, just darwin is ok, but being future ready.

Reply:
To clarify this point, "darwin" refers to the kernel that MacOS is built on so "darwin" should be the result obtained from `platform.system().lower()` for any Mac system. So, if anything, it's possible that it's too broad of an identifier, but I suggested this as the check assuming the issue to be fundamental to the MacOS infrastructure (i.e., the kernel)